test/integration_test.rb

```diff
@@ -0,0 +1,62 @@
+require 'helpers/kubeclient_helper'
+
+module KubernetesDeploy
+  class IntegrationTest < ::Minitest::Test
+    include KubeclientHelper
+
+    def setup
+      @logger_stream = StringIO.new
+      @logger = Logger.new(@logger_stream)
+      KubernetesDeploy.logger = @logger
+      @namespace = TestProvisioner.claim_namespace
+    end
+
+    def teardown
+      @logger_stream.close
+      TestProvisioner.delete_namespace(@namespace)
```

Comment:
I think you're right, they're run inside [`#capture_exceptions`](https://github.com/seattlerb/minitest/blob/master/lib/minitest/test.rb#L110) so they should complete.